bCNC/lib/bpath.py

```diff
@@ -1805,6 +1878,44 @@ def convert2Lines(self, minlen):
 			if segment.length()<=minlen:
 				segment.change2Line()
 
+	#----------------------------------------------------------------------
+	# Convert Arc to LINES  in a path
```

Comment:
There is already `def linearize()` why you need another method for the same thing???

Reply:
OK I did not see linearize function.
=> Can remove it.